src/pages/ProductDetail/components/Detail/TotalCost.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import './TotalCost.scss';
+
+const TotalCost = ({ price, count }) => {
+  return (
+    <div className="totalCost">
+      <span className="title">상품금액 합계 </span>
```

Comment:
```suggestion
      <span className="total amount">상품금액 합계 </span>
```
제목 보단, 총 금액이 어울릴것같습니다

Reply:
수정했습니다.